src/client/cmd/launch.cpp

```diff
@@ -220,3 +158,84 @@ mp::ParseCode cmd::Launch::parse_args(mp::ArgParser* parser)
 
     return status;
 }
+
+mp::ReturnCode cmd::Launch::request_launch()
+{
+    mp::AnimatedSpinner spinner{cout};
+
+    auto on_success = [this, &spinner](mp::LaunchReply& reply) {
+        spinner.stop();
+        cout << "Launched: " << reply.vm_instance_name() << "\n";
+        return ReturnCode::Ok;
+    };
+
+    auto on_failure = [this, &spinner](grpc::Status& status) {
+        spinner.stop();
+
+        if (status.error_code() == grpc::StatusCode::NOT_FOUND)
```

Comment:
No, this is triggered by the daemon seeing that the opt-in file has not been created and set and telling the client about it.  I used `NOT_FOUND` here since that was the most relevant grpc status enum I could find:)